### DISCUSSION TITLE
Multiplayer

### DISCUSSION THREAD

**Discussion by MadSochi** [Fetty42/FS22_RealisticAnimalLosses#2] [Q&A] [answered]
In multiplayer if nobody is online on the FARM with animals but other players in other Farms is online and time on the server is running while others players online - will animals on offline FARM without owner-players die ?

If YES - fix it. In other way this modification is useless in MultiPlayer.

**Comment by Fetty42** [comment, marked as answer]
Hi,
no, currently only animals on online FARMS die. 
The only bug was that animals not owned by a farm (FARM 0) die.
